cerebrium/environments/config-files.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Using Config Files
+description: Using config files to configure Cortex deployments easily and quickly
+---
+
+After you've created your Cortex project, you may find that you need more control over your deployment than the default `cerebrium deploy` command provides.
+For example, you may want to specify the number of GPUs to use, the amount of memory to use or even the version of Python for your environment.
+These settings and more are all configurable using config files.
+
+## Deployment Parameters
+
+Deployment parameters are govern the persistent environment in which your model is deployed.
+These parameters are specified under the `cerebrium.deployment` section of your config file.
+
+The available deployment parameters are:
+| parameter | description | type | default |
+| --- | --- | --- | --- |
+| `name` | The name of your deployment | string | my-model |
+| `python_version` | The Python version available for your runtime | float | 3.10 |
+| `include` | Local files to include in the deployment. | string | '[./\*, main.py]' |
+| `exclude` | Local Files to exclude from the deployment. | string | '[./.\*, ./__\*]' |
+
+## Build Parameters
+
+Build parameters are used during the build phase of your deployment.
```

Comment:
do we have some content about the phases or build phase? Could be nice to link to more info about that. Perhaps something to add in the future

Reply:
Good idea. 
We don't have anything yet. What would you want to include in that content?